web/src/features/fireWeather/components/graphs/WxDataGraph.tsx

```diff
@@ -51,13 +53,27 @@ const WxDataGraph = ({
   const noHighResModels = allHighResModelValues.length === 0
```

Comment:
Hmmm... maybe out of scope for this pr - but these variables should probably all have been names `haveThing` rather than `notHaveThing`; `!haveThing` and `haveThing` just read easier.

Reply:
Done!